### ISSUE TITLE
Documentation - HTTP Load balancer active service policies is marked as list, but must be only one policy

### ISSUE THREAD

**Issue by BobbyWinston** [volterraedge/terraform-provider-volterra#96]
The documentation shows that the item "Policies" is an ordered list : https://registry.terraform.io/providers/volterraedge/volterra/latest/docs/resources/volterra_http_loadbalancer#active-service-policies

But you have to set it up as a block : 

Working code : 

```hcl
  active_service_policies {
    policies {
      name      = "ves-io-allow-all"
      namespace = "shared"
      tenant    = "my-tenant"
    }
    policies {
      name      = "ves-io-next-policy-set"
      namespace = "shared"
      tenant    = "my-tenant"
    }
  }
```

Expected code according to documentation : 
```hcl
  active_service_policies {
    policies = [
    {
      name      = "ves-io-allow-all"
      namespace = "shared"
      tenant    = "my-tenant"
    },
    {
      name      = "ves-io-next-policy-set"
      namespace = "shared"
      tenant    = "my-tenant"
    }
    ]
  }
```

**Comment by cohenaj194** [comment]
The Terraform SDK used for this was originally designed for Terraform v0.11 and earlier and so it doesn't support configuration constructs that those older Terraform versions didn't support, and Terraform v0.11 and earlier didn't support lists of objects in the way you are intending here.

We can instead treat this like a documentation bug and specify that we need a list of policy blocks instead of an `An ordered list of references to service_policy objects`.

**Comment by BobbyWinston** [comment]
Yes, i would even say a map of policy blocks as you have to set it between `{ }` and not `[ ]` 
With an example added with multiple policies this should be enough.

Thank you for your answer
